### ISSUE TITLE
[Bug]: Update *.340 to *.341

### ISSUE THREAD

**Issue by mateusfmello** [coollabsio/coolify#3494]
### Error Message and Logs

Logs container `ghcr.io/coollabsio/coolify-helper:latest`

```
[root@cloud-001 rocky]# docker logs -f 4946d9393d45
 coolify Pulling 
 b2ddfd337773 Already exists 
 f6031115538d Already exists 
...
 bd98dda5264c Extracting [========================>                          ]   4.62MB/9.268MB
 bd98dda5264c Extracting [===============================================>   ]  8.749MB/9.268MB
 bd98dda5264c Extracting [==================================================>]  9.268MB/9.268MB
 bd98dda5264c Pull complete 
 coolify Pulled 
 Container XXXXXfc3b4a_coolify-db  Recreate
 Container XXXXXb194be_coolify-redis  Stopping
 Container XXXXXf05fb1_coolify-realtime  Recreate
 Container coolify-redis  Recreate
 Container XXXXXf05fb1_coolify-realtime  Recreated
 Container XXXXXfc3b4a_coolify-db  Recreated
 Container XXXXXb194be_coolify-redis  Stopped
 Container XXXXXb194be_coolify-redis  Removing
 Container XXXXXb194be_coolify-redis  Removed
Error response from daemon: Conflict. The container name "/XXXXXb194be_coolify-redis" is already in use by container "06e9be6651b1e5c974XXXXXXXXXXXXXXXf46b1035c758b7c0ae370ab9a4a9". You have to remove (or rename) that container to be able to reuse that name.
```

### Steps to Reproduce

Today I tried to update from version 340 to 341, but during the update process an error occurred causing the Coolify containers (Redis, PostgreSQL and Realtime) to have the status "Creating" in `docker ps`, only the Coolify container itself was started and the application gave an error connecting to PostgreSQL, since its container had not been started.

To solve the problem I ran the command `docker stop` and then `docker start` on the containers with the status "Creating", after that Coolify started working again, apparently normally, but I didn't do many tests.

However, it didn't update to version 341 and the update button disappeared.

I accessed the "Settings" screen and used the button to check for updates manually, the update button appeared again, I sent it to update and the error occurred again.

Apparently it is an error due to a conflict in the container names.

This time I was observing the logs of the "Coolify helper" container

### Coolify Version

v4.0.0-beta.341

### Are you using Coolify Cloud?

No (self-hosted)

### Operating System and Version (self-hosted)

Rocky Linux 9.4 (Blue Onyx)

**Comment by mateusfmello** [comment]
This is the error that occurs in the Coolify system, something normal, it does not resolve the DNS because the database is not working.

```
2024/09/18 21:50:39 [error] 137#137: *78437 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught PDOException: SQLSTATE[08006] [7] could not translate host name "postgres" to address: Name or service not known in /var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:66
Stack trace:
#0 /var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(66): PDO->__construct()
#1 /var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(85): Illuminate\Database\Connectors\Connector->createPdoConnection()
#2 /var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(48): Illuminate\Database\Connectors\Connector->tryAgainIfCausedByLostConnection()
#3 /var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connectors/PostgresConnector.php(35): Illuminate\Database\Connectors\Connector->createConnection()
#4 /var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connectors/ConnectionFactory.php(185): Illuminate\Database\Connectors\Pos...; PHP message: PHP Fatal error:  Uncaught PDOException: SQLSTATE[08006] [7] could not translate host name "postgres" to address: Name or service not known in /var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:66
```

**Comment by TomKaltz** [comment]
I have a similar issue.  I have a redis database in a project but now it's conflicting with the coolify-redis install.

**Comment by mateusfmello** [comment]
> I have a similar issue. I have a redis database in a project but now it's conflicting with the coolify-redis install.

1. Run `docker ps -a`
2. Retrieve the 4 IDs of the Coolify containers
3. Run `docker stop`
4. And finally run `docker start`

**Comment by mateusfmello** [comment]
I waited for a few more versions to be released and updated directly to `v4.0.0-beta.347` and the problem did not occur again.
